src/main/java/core/basesyntax/strategy/SupplyOperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperationHandler implements OperationHandler {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line.